App.js

```diff
@@ -1,11 +1,11 @@
 import { StatusBar } from 'expo-status-bar';
 import { StyleSheet, Text, View } from 'react-native';
+import Home from './app/Pages/Home'
 
 export default function App() {
   return (
     <View style={styles.container}>
```

Comment:
Something I came across during my reading is the <SaveAreaView>, it render content within the safe area boundaries of a device. You know that part of the device on newer iPhones that is black at the top of the screen? Some devices and androids have this and this component will make sure that your content will be placed below this on devices that have it. That way you won't have a lot of padding on the top of components that do have it. I think we should update this.